src/components/filter.jsx

```diff
@@ -130,7 +130,16 @@ class filter extends Component {
               <Dropdown.Item onClick={this.props.c12}>12 th</Dropdown.Item>
             </Dropdown.Menu>
           </Dropdown>
-
+        </div>
+        <div
+          style={{
```

Comment:
The styling here and at line 70 seems to be same. Would be nice if we can have this once in style.css file.